.github/workflows/bindings.yml

```diff
@@ -282,10 +281,9 @@ jobs:
         run: |
           bun x @puppeteer/browsers install chromedriver@latest --path $PWD
           echo "CHROMEDRIVER_PATH=$(echo $PWD/chromedriver/*/*/chromedriver)" >> $GITHUB_ENV
-      - name: build wasm artifacts
+      - name: build ts artifacts
         run: |
-          cd crypto-ffi
-          cargo make wasm
+          make ts
```

Comment:
This is going to build WASM in dev mode, right? I think we should do a release mode build in CI.